charts/shoot-core/components/charts/kube-proxy/values.yaml

```diff
@@ -1,16 +1,28 @@
-global:
-  podNetwork: 1.2.3.4/24
+vpaEnabled: true
+podNetwork: 1.2.3.4/24
 kubeconfig: dummy-b64-data-here
-kubernetesVersion: 1.15.2
 #featureGates: {}
 #  CustomResourceValidation: true
 #  RotateKubeletServerCertificate: false
 images:
-  kube-proxy: image-repository
   alpine: image-repository
 podAnnotations: {}
 
 enableIPVS: false
 
 ports:
   metrics: 10249
+
+workerPools:
+- name: ""
+  kubernetesVersion: 1.15.0
+  kubeProxyImage: image-repository:image-tag
+- name: cpu-worker
+  kubernetesVersion: 1.15.2
+  kubeProxyImage: image-repository:image-tag
+- name: cpu-worker
+  kubernetesVersion: 1.15.1
+  kubeProxyImage: image-repository:image-tag
+- name: cpu-worker2
+  kubernetesVersion: 1.14.8
+  kubeProxyImage: image-repository:image-tag
```

Comment:
Yes, right, just samples. I don't know if they should be removed. Usually, we specify such examples in all `values.yaml` files for all our Helm charts. Eventually, it does not matter too much since we will refactor the `kube-proxy` chart as part of #2754 and then all of this will be gone. If you want, I can remove or shrink this, but as said, it doesn't really matter.

Reply:
Sure, no need to change this, I was under the impression that values added here should be treated as "defaults" rather than "samples", and I think in this chart there is a mixture of both - a bit confusing, perhaps, but certainly very minor.